source/src/processing/cs_CommandHandler.cpp

```diff
@@ -86,81 +86,8 @@ void CommandHandler::handleCommand(
 		return;
 	}
 
-	switch (type) {
```

Comment:
I see your point that allowAccess() already checks if the control command type exists, maybe add a comment that it does.
This switch case can be replaced by:
```
switch (type) {
case CTRL_CMD_SET_SUN_TIME:
break;
default:
LOGd("cmd=%u lvl=%u", type, accessLevel);
break;
}

Reply:
nice solution :-)